src/components/Form/Autocomplete.tsx

```diff
@@ -18,12 +21,49 @@ const Autocomplete = ({
     onChange,
     dataCy,
     dataLoader,
+    type,
     mode = "multiple",
     style = {},
     value = [],
     preloadedOptions = [],
     preloadedTopics = [],
 }: AutocompleteProps) => {
+    const [treatedValue, setTreatedValue] = useState([]);
```

Comment:
**issue(blocking):** this logic should be in a input wrapper just for user. I suggest you create a new input called `UserInput` and add this logic there, and touch the autocomplete component as little as possible.